packages/govtnz-ds-website/src/commons/Example.tsx

```diff
@@ -52,7 +53,7 @@ export default class Example extends Component<Props, State> {
       hasClickedExpand: false,
       copyingMode: false,
       supportsJavaScript: false,
-      supportsClipboard: false,
+      supportsClipboard: true,
```

Comment:
If JavaScript is disabled does this mean that the button is server rendered by Gatsby? We might want to set this state in a react lifecycle that runs clientside.